engine/src/main/resources/templates/Rbac/platformGroup2customEmailBody.html

```diff
@@ -0,0 +1,52 @@
+{#include Rbac/insightsEmailBody}
+{#content-title}Rbac platform group Notification - {action.timestamp.toStringFormat()}{/content-title}
+{#content-body}
+<tr>
+    <td class="rh-body rh-multi-column">
+        <table class="rh-content" role="presentation" align="center" border="0" cellpadding="0" width="100%">
+            <tbody>
+            <!-- Content block -->
+            <tr>
+                <td>
+                    <!-- end 50 col -->
+                    <!--[if (gte mso 9)|(IE)]>
+                    </td>
+                    </tr>
+                    </table>
+                    <![endif]-->
+                </td>
+            </tr>
+            <!-- end content block -->
+            </tbody>
+        </table>
+    </td>
+</tr>
+<!-- end body section -->
+
+<!-- Body section -->
+<tr>
+    <td class="rh-body">
+        <table class="rh-content" role="presentation" align="center" border="0" cellpadding="0" cellspacing="0" width="100%">
+            <tbody>
+            <!-- Content block -->
+            <tr>
+                <td class="rh-content__block">
+                    <p>
+                        Hi,
+                        <br>
+                        <br>
+                        Platform default group is modified and the user is responsible for managing it from now on.
+                        <br>
+                        Check the custom platform defualt group here: <a class="rh-url" href="https://console.redhat.com/api/rbac/v1/groups/{action.events[0].payload.uuid}/"><b>{action.events[0].payload.name}</b></a>{/if}
```

Comment:
```suggestion
                        Check the custom platform default group here: <a class="rh-url" href="https://console.redhat.com/api/rbac/v1/groups/{action.events[0].payload.uuid}/"><b>{action.events[0].payload.name}</b></a>{/if}
```

Reply:
Is this URL pointing directly to the API intended?

Reply:
Yea, so it could show the customer what it looks like. Any concerns with it?

Reply:
I updated with UI link, which might be better. Thanks!

Reply:
That's what I meant to ask here, thanks!